gallery.html

```diff
@@ -2,10 +2,67 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="author" content="Juan Nunez" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Gallery</title>
+    <!--Font Links Section-->
+    <link rel="preconnect" href="https://fonts.gstatic.com"> 
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.gstatic.com"> 
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap" rel="stylesheet">
+    <!--CSS Links Section-->
+    <link rel="stylesheet" href="index.css">
+
+    <title>Juan's Gallery</title>
 </head>
 <body>
+    <header>
+        <nav class="topnav">
+            <ul class="navul">             
+                <li class="navli"><a href="index.html">Home</a></li>|
+                <li class="navli"><a href="gallery.html">Gallery</a><li>|
+                <li class="navli"><a href="contact.html">Contact me</a></li>
+            </ul>
+        </nav>
+    </header>
+    
+    <main>
+        <center><h1>My favorite collection of cars</h1></center>
+
+        <div class="gallery">
+            <center>
+            <div class="column"><img src="images/A7.jpg" alt="Picture of an Audi A7" width="200"></div>
+            <div class="column"><img src="images/c10.jpg" alt="Picture of a silverado c10" width="200"></div>
+            <div class="column"><img src="images/impala.jpg" alt="Picture of an old school impala " width="200" ></div>
+            </center>
+        </div>
+        <br><br>
+        <div class="reset"></div>
+
+        <div class="gallery">
+            <center>
+            <div class="column"><img src="images/mytruck.jpg" alt="Picture of my 2015 silverado" width="200" height="250"></div>
+            <div class="column"><img src="images/GTR.jpg" alt="Picture of a GTR" width="200"></div>
+            <div class="column"><img src="images/oldschoolchallenger.jpg" alt="Picture of an old school challenger" width="200" height="250"></div>
+            </center>
+        </div>
+    </main>
+    <div class="reset"></div>
+    <br>
+
+    <footer>
+
+        <div class="footernav">
+    
+                <ul class="footerul">
+                        <li class="footerli"><a href="https://www.linkedin.com/in/juannunez1129" target="_blank">LinkdIn</a></li>
+                        <!--Porfolio is offline need to find a new hosting site-->
+                        <li class="footerli"><a href="#">Porfolio</a></li>
+                        
+                        <li class="footerli"><a href="images/JuanNunezResume.pdf" target="_blank">Resume</a><li>    
+                </ul>
+        </div>
+    </footer>
```

Comment:
This footer nav appears to be a duplication of code found in index.html (as does the "header" menu).  Consider a refactoring to replace this duplication, by encapsulating what changes (i.e. encapsulate the gallery.html code from the duplicate code, by moving the duplicate code to another file -- this will obviously require a little research on your part to learn how to display the contents of a separate HTML file here in the gallery.html file).